Documentation/Configuration.md

```diff
@@ -118,6 +118,18 @@ You can also run this command to see the list of rules in the default
 
     $ swift-format dump-configuration
 
+## Global Configuration
+
+If no `.swift-format` can be found for the current project/file, the configuration directories
+are searched for a `swift-format/config.json` file. While the filename is different,the
+configuration format stays the same.
+
+Locations that are searched, in this order:
+
+- `~/Library/Application Support/swift-format/config.json`
+- `$XDG_CONFIG_HOME/swift-format/config.json`
+- `~/.config/swift-format/config.json`
```

Comment:
I'd love to see $XDG_CONFIG_HOME / ~/.config first here. What's the thinking behind looking first in ~/Library/Application Support?

Reply:
It is the default path for configuration files on MacOS